.github/workflows/tests.yml

```diff
@@ -30,7 +30,8 @@ jobs:
             libpq-dev libsqlite3-dev \
             libhiredis-dev \
             libmongoc-dev \
-            libmicrohttpd-dev
+            libmicrohttpd-dev \
+            liblog4cplus-dev
```

Comment:
I don't see how this change, and the change to Dockerfile, have anything to do with the change to the `run_tests.sh` program?